src/sentry/tasks/auto_review_inbox.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.models import GroupInbox, Organization, Project, OrganizationStatus
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(
+    name="sentry.tasks.schedule_auto_review_inbox", time_limit=65, soft_time_limit=60
+)
+def schedule_auto_review_inbox():
+    for organization in Organization.objects.filter(status=OrganizationStatus.ACTIVE).iterator():
+        for project in Project.objects.filter(organization=organization).values("id"):
+            auto_review_inbox.delay(project["id"])
```

Comment:
Just to put this into perspective now, every 15 minutes, we're going to be scheduling 10s of thousands of jobs for projects that likely have seen no data in this time period, etc. This isn't what I suggested to do at all.

Reply:
Without exposing the numbers, the order of magnitude right now is tens of millions of rows are in this bucket for deletion on the first pass. In the future, it'll be on the magnitude of how many groups are generated per interval (15 minutes as of now). So it'll definitely be more manageable moving forward once we get the initial trim of 10s of millions of rows out of the way.

Either way though, this is more suited for using our `sentry.db.deletion:BulkDeleteQuery` handler just based on the magnitudes we're talking here.

I'd still want to probably kick off something manually for the first initial truncating since it's just so disproportionately large relatively to the ongoing maintenance.

Reply:
> Just to put this into perspective now, every 15 minutes, we're going to be scheduling 10s of thousands of jobs for projects that likely have seen no data in this time period, etc. This isn't what I suggested to do at all.

Also my bad here, this is actually nearly 1 million. Not 10s of thousands.

Reply:
I agree that scheduling more jobs isn't the right answer. Thanks for the `entry.db.deletion:BulkDeleteQuery` -- I didn't know that was a thing.